extension/js/scripts.js

```diff
@@ -34,6 +33,9 @@ function package_view(dir) {
 
         $('#packageview').show();
         $('#loadingview').hide();
+    }, function () {
+        common.display_msg("Server Error", "The project configuration could not be retrived. Please try again.");
```

Comment:
How can I try again? Closing and clicking again in the cog?

Reply:
Yes. Is that okay?

Reply:
Ok, let's leave it as is. I'm gonna merge and test everything again and create an issue with my observations, ok?